.mise.toml

```diff
@@ -0,0 +1,5 @@
+[env]
+NODE_OPTIONS = "--openssl-legacy-provider"
```

Comment:
This is unrelated to this PR but needed to build the browser dist by running `npm run build`. I can remove it from this PR if the reviewer prefers.